### PR TITLE
Add repetition args for inscriptions

### PR DESCRIPTION
**Pull request by victorkirov**
# 🔘 PR Type

What kind of change does this PR introduce?

- [x] Enhancement

# 📜 Background

Allow for repeat inscription mints

# 🔄 Changes

Does this PR introduce a breaking change?

- [ ] Yes, Incompatible API changes
- [x] No, Adds functionality (backwards compatible)
- [ ] No, Bug fixes (backwards compatible)

Changes:

- Switch to using v2 endpoint for inscriptions
- Adds the repetitions field and uses the totalInscriptionValue field from the response

Impact:

- It's backwards compatible with the previous implementation, so it can be updated in place
- For repeat inscriptions, the caller will need to pass in the optional `repetitions` arg

# ✅ Review checklist

Please ensure the following are true before merging:

- [ ] Code Style is consistent with the project guidelines.
- [ ] Code is readable and well-commented.
- [ ] No unnecessary or debugging code has been added.
- [ ] Security considerations have been taken into account.
- [x] The change has been manually tested and works as expected.
- [ ] Breaking changes and their impacts have been considered and documented.
- [ ] Code does not introduce new technical debt or issues.

### CODE REVIEW

File: transactions/inscriptionMint.ts [hunk 1]
```diff
@@ -244,6 +253,7 @@ export async function inscriptionMintExecute(executeProps: ExecuteProps): Promis
     availableUtxos: addressUtxos,
     feeRate,
     network,
+    useUnconfirmed: false,
```
Comment (victorkirov) [inline transactions/inscriptionMint.ts:256]: Note this change. We are no longer allowing the use of unconfirmed UTXOs as inputs for inscription txns.

File: types/api/xverseInscribe/inscription.ts [hunk 2]
```diff
@@ -35,6 +40,7 @@ export type InscriptionCreateOrderResponse = {
   commitValue: number;
   commitValueBreakdown: {
     inscriptionValue: number;
+    totalInscriptionValue: number;
```
Comment (teebszet) [inline types/api/xverseInscribe/inscription.ts:43]: @victorkirov does this get passed to the hook return values?

**Comment by teebszet** [comment]
can't verify with manual test yet...

File: types/api/xverseInscribe/inscription.ts (hunk 2 above)
Comment (victorkirov) [reply types/api/xverseInscribe/inscription.ts:43]: It was not. Good catch 👍  😄

**Pull request merged**